solver/src/reasoners/cp/mod.rs

```diff
@@ -384,3 +406,87 @@ impl Backtrack for Cp {
 //         BindingResult::Unsupported
 //     }
 // }
+
+/* ========================================================================== */
+/*                                    Tests                                   */
+/* ========================================================================== */
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    /* ================================= Sum ================================ */
+
+    /// Returns the constraint `x + 2 <= 0`
+    fn simple_constraint() -> (LinearSumLeq, Domains) {
```

Comment:
Probably missing some tests with literals other that `TRUE`. 
Checking that you get the expected explanation is also important.

Reply:
Also with negative/null factors

Reply:
The code coverage has been increased in the last commits